source/_layouts/open_source.html

```diff
@@ -16,11 +16,54 @@
+    {% for project in site.oss_projects %}
+      {% if project["category"] == "featured" %}
+        {% capture project %}{{ project }}{% endcapture %}
+        {% include oss_page_project.html %}
+      {% endif %}
+    {% endfor %}
+  </section>
+  <section id="javascript" class="open-source-category">
+    <div class="open-source-category-header"><div class="open-source-category-header-title"><span><b>Title</b></span></div><div class="open-source-category-header-description"><span><b>Description</b></span></div><div class="open-source-category-header-updated"><span><b>Created</b></span></div><footer></footer></div>
+    {% for project in site.oss_projects %}
+      {% if project["category"] == "javascript" %}
+        {% capture project %}{{ project }}{% endcapture %}
+        {% include oss_page_project.html %}
+      {% endif %}
+    {% endfor %}
+  </section>
+  <section id="ios" class="open-source-category">
+    <div class="open-source-category-header"><div class="open-source-category-header-title"><span><b>Title</b></span></div><div class="open-source-category-header-description"><span><b>Description</b></span></div><div class="open-source-category-header-updated"><span><b>Created</b></span></div><footer></footer></div>
+    {% for project in site.oss_projects %}
+      {% if project["category"] == "ios" %}
+        {% capture project %}{{ project }}{% endcapture %}
+        {% include oss_page_project.html %}
+      {% endif %}
+    {% endfor %}
+    
+  </section>
+  <section id="ruby" class="open-source-category">
+    <div class="open-source-category-header"><div class="open-source-category-header-title"><span><b>Title</b></span></div><div class="open-source-category-header-description"><span><b>Description</b></span></div><div class="open-source-category-header-updated"><span><b>Created</b></span></div><footer></footer></div>
+    {% for project in site.oss_projects %}
+      {% if project["category"] == "ruby" %}
+        {% capture project %}{{ project }}{% endcapture %}
+        {% include oss_page_project.html %}
+      {% endif %}
+    {% endfor %}
+    
+  </section>
+  <section id="devops" class="open-source-category">
+    <div class="open-source-category-header"><div class="open-source-category-header-title"><span><b>Title</b></span></div><div class="open-source-category-header-description"><span><b>Description</b></span></div><div class="open-source-category-header-updated"><span><b>Created</b></span></div><footer></footer></div>
+    {% for project in site.oss_projects %}
+      {% if project["category"] == "devops" %}
+        {% capture project %}{{ project }}{% endcapture %}
+        {% include oss_page_project.html %}
+      {% endif %}
+    {% endfor %}
+    
```

Comment:
Couldn't really think of a super elegant way to do this, so just opted for the simplest.